node/runtime/src/lib.rs

```diff
@@ -424,6 +425,33 @@ impl finality_tracker::Trait for Runtime {
 	type ReportLatency = ReportLatency;
 }
 
+impl system::offchain::CreateTransaction<Runtime, UncheckedExtrinsic> for Runtime {
```

Comment:
where is this being used?

Reply:
We pass this as `SubmitTransaction` to `im-online`:
```
type SubmitTransaction = TransactionSubmitter<ImOnlineId, Runtime, UncheckedExtrinsic>;
```
(see `Runtime` as the second parameter)